src/ansible_navigator/configuration_subsystem/navigator_configuration.py

```diff
@@ -375,6 +375,15 @@ class Internals:
             subcommands=["inventory", "run"],
             value=SettingsEntryValue(),
         ),
+        SettingsEntry(
+            name="json_schema",
+            choices=[True, False],
+            cli_parameters=CliParameters(short="--js", action="store_true"),
+            settings_file_path_override="settings.json-schema",
+            short_description="Generate a json schema for the settings file",
```

Comment:
Can also mention it generates draft7 schema

Reply:
done